src/veh_interact.cpp

```diff
@@ -18,6 +18,8 @@
 #define ISNAN std::isnan
 #endif
 
+static const std::set< ammotype > unsiphonable = { "gasoline", "diesel", "battery", "plut_cell", "plasma" };
```

Comment:
Wait, can I really not siphon the few precious fuel charges? That is one of the main fuel sources. Or does this just mean un-*drain*-able?

Actually what is the difference between drain and siphon? Afaik, up to now siphon applies to diesel/gasoline and drain applies to water. If that's the only difference, it should be combined into one action that applies to all liquids, including generic ones.